### PR TITLE
Update docs for categories and Pso types based on discussion with Abhijit

### PR DESCRIPTION
**Pull request by DmitryLanda**

### CODE REVIEW

File: pso_types.md [hunk 1]
```diff
@@ -85,9 +87,45 @@ Response: `200 OK`
...
 JSON structure for both errors will be same we have in v3.2
+
+Changes in other API endpoints
+------------------------------
+
+#### Field Creation
+To handle PSO Type we will add new property in field creation request 
+- `pso_type` - string, required. PSO Type alias (trigram)
+
+**Please note: PSO Type cannot be changed if field created**
```
Comment (abhigokhale) [inline pso_types.md]: Just to confirm, we are saying once the field is created while editing the field the PSO Type associated with the field cannot be changed.

File: pso_types.md [hunk 2]
```diff
@@ -85,9 +87,45 @@ Response: `200 OK`
...
+ 
+In both API  we will add new optional parameter:
+ - `pso-type` - string, optional. PSO Type alias (trigram). If presented - only forms /instances assigned to selected PSO Type will be presented
+
+#### Form Creation
+To handle PSO Type we will add new property in form creation request 
+- `pso_type` - string, required. PSO Type alias (trigram)
+
+**Please note:** 
+ - PSO Type cannot be changed if form created**
```
Comment (abhigokhale) [inline pso_types.md]: Just to confirm, we are saying once the form is created while editing the form the PSO Type associated with the form cannot be changed.

File: pso_types.md [hunk 3]
```diff
@@ -85,9 +87,45 @@ Response: `200 OK`
...
+ - Only fields that belongs to selected PSO Type can be attached to form
+
```
Comment (abhigokhale) [inline pso_types.md]: Please refer 'Assignment Creation' section present on the Page 29 of the MPRH Spec V3.3 - Build 1 document. Do we need to add API for getting options in the Target Population and Workflow section while creating form assignment?

Comment (DmitryLanda) [reply pso_types.md]: We will have API to get list of available options for WF. Will add later, along or may be after PSO

File: pso_types.md (hunk 2 above)
Comment (DmitryLanda) [reply pso_types.md]: yes - will provide JSON for PSO (form) later in separate commit

File: pso_types.md (hunk 1 above)
Comment (DmitryLanda) [reply pso_types.md]: confirmed with @obrahimi and @riadlasfer

**Pull request merged**